tests/functional/spec/theme_northernireland.spec.js

```diff
@@ -0,0 +1,22 @@
+import RadioPage from "../generated_pages/theme_northernireland/radio.page";
+import SubmitPage from "../generated_pages/theme_northernireland/submit.page.js";
+
+describe("Theme Northern Ireland", () => {
+  describe("Given I launch a Northern Ireland themed questionnaire", () => {
+    before(() => {
+      browser.openQuestionnaire("test_theme_northernireland.json");
+    });
+
+    it("When I navigate to the radio page, Then I should see Northern Ireland theme content", () => {
+      expect(browser.getUrl()).to.contain(RadioPage.pageName);
+      expect($("#ni-finance-logo-alt").isExisting()).to.equal(true);
+      expect($("#ni-finance-logo-alt").getHTML()).to.contain("Northern Ireland Department of Finance logo");
```

Comment:
How stable is using ni-finance-logo-alt as it comes from the design system?

Reply:
We could potentially look for the svg logo's class which exists in all themes (to make it more general) and then it's html contents? Some mix of these assertions:
```
expect(browser.getUrl()).to.contain(RadioPage.pageName);
expect($(".ons-svg-logo").isExisting()).to.equal(true);
expect($(".ons-svg-logo").getHTML()).to.contain("ni-finance-logo-alt").to.contain("Northern Ireland Department of Finance logo");;
expect($(".ons-svg-logo").getHTML()).to.contain("Northern Ireland Department of Finance logo");
```

Reply:
NI doesn't differ much from other themes in runner:
```
@dataclass
class NorthernIrelandBusinessSurveyConfig(BusinessSurveyConfig):

    page_header_logo: str = "ni-finance-logo"
    page_header_logo_alt: str = lazy_gettext(
        "Northern Ireland Department of Finance logo"
    )
    mobile_logo: str = "ni-finance-logo-mobile"
    custom_header_logo: bool = True
```

Reply:
We pin DS, so it's not a disaster, but looking at the macro I see this ```        <svg class="ons-svg-logo" xmlns="http://www.w3.org/2000/svg" focusable="false" width="296" height="56" viewBox="0 0 592 112" aria-labelledby="ni-finance-logo-alt">
            <title id="ni-finance-logo-alt">{{ params.altText | default ('Northern Ireland Department of Finance logo')}}</title>``` 
So the only thing we potential are of control of is params.altText

Reply:
Unfortunately are value is the same as the default 😆

Reply:
Possibly we can:
`expect($(".ons-svg-logo").getHTML()).to.contain("aria-labelledby=\"ni-finance-logo-alt\"");`